js/seedrandom.js

```diff
@@ -0,0 +1,20 @@
+import seedrandom from 'seedrandom'
+
+const SEED = 'PlateTectonics3D'
+let rand = null
+
+export function initialize (deterministic) {
+  rand = seedrandom(SEED, { state: true, entropy: !deterministic })
```

Comment:
I had to look through the `seedrandom` readme to understand these constructors - it might be worth adding a couple of comments to clarify the parameters

Reply:
Thanks, I'll do it.